### PR TITLE
VB-2640/VB-2641/VB-2563 Multiple prisons - block dates

### PR DESCRIPTION
**Pull request by hutcheonb-moj**
## Description
Block dates for the following prisons:
DMI - CFI - SHI

## Types of changes
- [x] Temporary timetable change

### CODE REVIEW

File: db/seeds/prisons/DMI-durham.yml [hunk 1]
```diff
@@ -37,9 +37,9 @@ anomalous:
...
 unbookable:
-- 2023-07-31
 - 2023-08-17
 - 2023-09-25
+- 2023-09-27
 - 2023-10-30
 - 2023-11-27
-- 2023-12-18
\ No newline at end of file
+- 2023-12-18
```
Comment (AledWEvans) [inline db/seeds/prisons/DMI-durham.yml:45]: what this one it's on the ticket?

**Review by AledWEvans — COMMENTED** [review]
I have a question, see comment

File: db/seeds/prisons/DMI-durham.yml (hunk 1 above)
Comment (hutcheonb-moj) [reply db/seeds/prisons/DMI-durham.yml:45]: added newline at EOF - view inline and you can see it's removed + re-added

**Review by AledWEvans — APPROVED** [review]
LGTM

**Pull request merged**